Codecs/DAG-PB.md

```diff
@@ -10,10 +10,10 @@ The DAG-PB IPLD format is a legacy format implemented with a single protobuf.
 // An IPFS MerkleDAG Link
 message PBLink {
 
-  // multihash of the target object
+  // binary CID (with no multibase prefix) of the target object
```

Comment:
Would "binary version 0 CID (no multibase prefix) of the target object" be accurate? It might be nice to be explicit about why this is odd.

Reply:
Any CID can go here. The key part is that we're not including the multibase (which we sometimes (e.g., CBOR) do include in binary).